.storybook/main.js

```diff
@@ -1,8 +1,8 @@
 module.exports = {
   stories: [
-    '../src/stories/*.stories.mdx',
-    '../src/stories/*.stories.@(js|jsx|ts|tsx)',
-    '../src/stories/*/*.stories.@(js|jsx|ts|tsx)',
+    './stories/*.stories.mdx',
+    '../src/components/*/*.stories.mdx',
+    '../src/components/*/src/*.stories.@(js|jsx|ts|tsx)'
```

Comment:
Do we actually have files residing at a location identified by this regex?

Reply:
3 and 5 are used. 4 is not, I will remove it